api/src/pcapi/core/offers/repository.py

```diff
@@ -740,3 +740,15 @@ def exclude_offers_from_inactive_venue_provider(query: flask_sqlalchemy.BaseQuer
 def get_next_product_id_from_database() -> int:
     sequence: sa.Sequence = sa.Sequence("product_id_seq")
     return db.session.execute(sequence)
+
+
+def has_active_offer_with_ean_or_isbn(ean: str | None, isbn: str | None, venue: offerers_models.Venue) -> bool:
+    if ean is not None:
+        extra_data_filter = models.Offer.extraData["ean"].astext == ean
+    else:
+        extra_data_filter = models.Offer.extraData["isbn"].astext == isbn
+    return db.session.query(
+        models.Offer.query.filter(
+            models.Offer.venue == venue, models.Offer.isActive.is_(True), extra_data_filter
+        ).exists()
+    ).scalar()
```

Comment:
Haha je me sens ignoré :stuck_out_tongue: 

C'était du medium logaf donc pas grave. 


<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
Mais du coup **❔ question:** ‏que se passe-t-il si les 2 champs sont None ? On veut peut être logguer notre mécontentement ?

Reply:
si on a pas d'isbn ni d'ean on ne fait pas ce check -> https://github.com/pass-culture/pass-culture-main/commit/80d6ad1a6456c0fc64530acfb731a4079115e8ff

Reply:
Justement du coup, si ça arrive c'est annormal, on préfère le savoir dans Sentry plutôt que ce soit silencieux
Qui sait qui réutilisera cette fonction plus tard, et comment

Reply:
Ah yes je voit, j'ai modifié ! Par contre je suis pas sûr de voir ce que j'ai ignoré

Reply:
Ah ^^ 
[Ce comm](https://github.com/pass-culture/pass-culture-main/pull/5791#discussion_r1145906184)

Tu peux faire 
```python
extra_data_filter = models.Offer.extraData["ean"].astext == ean if ean is not None else Offer.extraData["isbn"].astext == isbn
```